### PR TITLE
ACP2E-411: [Documentation] Mention that Not Visible Individually products can be reordered

### PR DESCRIPTION
**Pull request by vkolesny**
## Purpose of this pull request

_Describe the goal and the type of changes this pull request covers. Tell us what changes you are making and why._

According to the Product Owner clarification:
https://jira.corp.magento.com/browse/ACP2E-204?focusedCommentId=1759838&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1759838
Please mention in the callout that Not Visible Individually products can be reordered.

## Magento release version

_Which Magento release(s) are affected by the content changes: 2.4, 2.3? Specify a patch release number, if applicable._

Magento 2.4.x

## Product editions

_Is this update specific to a product edition: Magento Open Source only, Adobe Commerce only?_

Both Magento Open Source and Adobe Commerce

_Is this update specific to an installed feature extension: B2B extension, other feature set (please, specify)?_

No

## Affected documentation pages

_List HTML links for affected pages on <https://docs.magento.com/user-guide/>._

https://docs.magento.com/user-guide/customers/account-dashboard-reorders.html
https://docs.magento.com/user-guide/sales/reorders-allow.html

### CODE REVIEW

File: src/customers/account-dashboard-reorders.md [hunk 1]
```diff
@@ -22,6 +22,9 @@ Customer is redirected to the [shopping cart]({% link quick-tour/shopping-cart.m
 
 **Case 2.** Some/all products from the order are not available for reorder
 
+{:.bs-callout-info}
+It is possible to reorder `Not Visible Individually` products
```
Comment (dobooth) [inline src/customers/account-dashboard-reorders.md:26]: Please add a period at the end of the sentence for each instance.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/customers/account-dashboard-reorders.md (hunk 1 above)
Comment (vkolesny) [reply src/customers/account-dashboard-reorders.md]: Done

**Review by dobooth — APPROVED** [review]

**Comment by m2-github-services** [comment]
@dobooth an error occurred during the Pull Request import.

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by m2-github-services** [comment]
@dobooth the branch with the code is successfully imported into the `magento-devdocs/merchdocs` repository. Branch name: `imported-magento-l3-merchdocs-1652`.

**Pull request merged**